src/main/java/pl/gov/coi/pomocua/ads/jobs/JobsResource.java

```diff
@@ -0,0 +1,34 @@
+package pl.gov.coi.pomocua.ads.jobs;
```

Comment:
One more thing: I see that currently api files are in `pomocua/ads/dictionaries/api/`. Maybe we should move it accordingly?

Reply:
Guys that did the dictionaries just came to the conclusion that they are going to drop the `api` package as the domain is so small

Reply:
Ok, good to know. Will they refactor their endpoint?